packages/jest-runtime/src/vendor/writeFileAtomic.js

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable */
```

Comment:
CI is complaining that this file doesn't have the FB copyright header. Should it? I'm _really_ not familiar with how these things work

Reply:
Copying write-file-atomic's ISC license into it in addition to your comment would be the safest thing IMO. Regarding the header, take a look at https://github.com/facebook/jest/blob/a551b19fbee3e556c6ae5e3a2c6c5ac78a0f8eb9/scripts/checkCopyrightHeaders.js#L80

Reply:
updated 🙂